.buildkite/pipeline_trigger.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env sh
+
+if [[ "$BUILDKITE_MESSAGE" == *"[barebones ci]"* ]]; then
+  echo "Running barebones build due to commit messages"
+  echo "Unit and static tests will be run"
+  echo "Minimal instrumentation tests will be run"
+  echo "End-to-end smoke tests will be run on minimum and maximum supported Android versions"
+elif [[ "$BUILDKITE_MESSAGE" == *"[full ci]"* ||
+  "$BUILDKITE_BRANCH" == "next" ||
+  "$BUILDKITE_BRANCH" == "master" ]]; then
+  echo "Running full build"
+  echo "Unit and static tests will be run"
+  echo "All instrumentation tests will be run"
+  echo "All end-to-end tests will be run on all supported Android versions"
+  buildkite-agent pipeline upload .buildkite/pipeline.full.yml
```

Comment:
I've just realised (having asked for a `[full ci]` that the post-merge build will not complete, because the full pipeline has dependencies on steps that have now moved to the quick file (which isn't loaded now).

Can we simply add the quick pipeline here for now and create a fresh ticket to have another look?